### PR TITLE
:sparkles: expense / image file component

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

**Comment by guillaumecrespel** [comment]
@delphinemillet  et @fabienjuif it's so bad ?? 😶

File: ui/ui-expenses/src/components/file/file.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './file.styles'
+
+const File = ({ name, id }) => (
+  <a href={`https://drive.google.com/uc?export=view&id=${id}`}>
+    <img className={styles.img} src={`https://drive.google.com/uc?export=view&id=${id}`} alt={name} />
```
Comment (delphinemillet) [inline ui/ui-expenses/src/components/file/file.jsx]: alt attribute not really useful.
Is it a need for you to see the uploaded image ? In first version, the image was hidden.
If this is needed, we have to think about a way to include it in the interface.

Comment (fabienjuif) [reply ui/ui-expenses/src/components/file/file.jsx]: btw, I think this is not a V1 feature

**Comment by guillaumecrespel** [comment]
I let this PR atm, just to information.

**Pull request closed without merging**